atlas/templates/settings.php

```diff
@@ -44,6 +44,12 @@
 {% endif %}
 $conf['googleanalytics_account'] = 'UA-25752450-1';
 
+{% if google_tag_client_container_id %}
+  {% if environment == 'local', 'dev', 'test' -%}
```

Comment:
Not doing a `in list` comparison, this only checks the to see if the field is equal to `local `, not sure what happens with the other 2 args.
```suggestion
  {% if environment in ['local', 'dev', 'test'] -%}
```